services/discovery/tox.ini

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-present, CloudZero, Inc. All rights reserved.
+# Licensed under the BSD-style license. See LICENSE file in the project root for full license information.
+
+[pytest] # This is for when you're just running py.test directly.
+addopts =
+    --doctest-modules
+    --showlocals
+    --ignore=setup.py
+    -vvv
+python_files = test_*.py !check_*.py !legacy_*.py
+test_paths = test
+norecursedirs=.git .tox .cache .py36* .aws-sam
+                # ^ NO TRAILING SLASHES ON DIRECTORIES!!
+
```

Comment:
is this even used?  Looked like we were running tests via pytest directly (generally not opposed to this given prior conversation)

Reply:
Yes. `pytest` and `flake8` honor this file. We discussed this before when we created the `interview-pairing` project. 😉

Reply:
I guess I was just confused by all the duplicate makefiles